.NET/Microsoft.Recognizers.Text.DateTime/Extractors/BaseDurationExtractor.cs

```diff
@@ -220,26 +220,11 @@ private List<ExtractResult> MergeMultipleDuration(string text, List<ExtractResul
                 return extractorResults;
             }
 
-            // If the first and last extractions are both preceded/followed by modifiers,
-            // they should not be merged, e.g. "last 2 weeks and 3 days ago"
-            var minStart = extractorResults.Min(er => er.Start);
-            var beforeStr = text.Substring(0, (int)minStart);
-            var beforeMod = this.config.ModPrefixRegex.MatchEnd(beforeStr, trim: true);
-            if (beforeMod.Success)
-            {
-                var maxEnd = extractorResults.Max(er => er.Start + er.Length);
-                var afterStr = text.Substring((int)maxEnd);
-                var afterMod = this.config.ModSuffixRegex.MatchBegin(afterStr, trim: true);
-                if (afterMod.Success)
-                {
-                    return extractorResults;
-                }
-            }
-
             var unitMap = this.config.UnitMap;
             var unitValueMap = this.config.UnitValueMap;
             var unitRegex = this.config.DurationUnitRegex;
             List<ExtractResult> ret = new List<ExtractResult>();

```

Comment:
Let's rename "ret" and "separateRet" for clarity. Maybe "mergedResults" and "separateResults"?

Reply:
renamed as 'results' and 'separateResults' (since 'results' at the end will contain both merged and unmerged results)